examples/relay_feedback_system.py

```diff
@@ -78,18 +104,40 @@ def plot_system(X_sim, t_grid):
     ax.set_ylabel("$x_2$")
     ax.set_zlabel("$x_3$")
     ax.grid()
+    plt.show()
+
+def plot_system(results):
+    nosnoc.latexify_plot()
+    X_sim = results["X_sim"]
+    t_grid = results["t_grid"]
 
     # state trajectory plot
     plt.figure()
-    plt.subplot(1, 2, 1)
-    plt.plot(t_grid, x1)
-    plt.plot(t_grid, x2)
-    plt.plot(t_grid, x3)
-    plt.xlabel("$t$")
-    plt.ylabel("$x(t)$")
+    for i in range(NX):
+        plt.subplot(1, NX, i+1)
+        plt.plot(t_grid, [x[i] for x in X_sim])
+        plt.grid()
+        plt.xlabel("$t$")
+        plt.ylabel(f"$x_{i+1}(t)$")
+
+    # algebraic variables
+    plt.figure()
+    plt.subplot(2, 1, 1)
+    lambdas = [results["lambda_sim"][0][0]
```

Comment:
Ahhh, this formatting is a little wierd but I presume its for line length reasons?

Reply:
yes.